lib/run_part.c

```diff
@@ -24,7 +24,7 @@ static int run_part (char *script_path, const char *name, const char *action)
 
 	pid=fork();
 	if (pid==-1) {
-		fprintf (shadow_logfd, "Could not fork: %s\n", strerror(errno));
+		fprintf (shadow_logfd, "fork: %s\n", strerror(errno));
```

Comment:
Can you add `failed` at the end to make it more understandable when reading the logs? Something like `fork failed: %s...`

Reply:
The format used here is common in many programs:

```sh
$ cat /nonexistent
cat: /nonexistent: No such file or directory
```

The errno description already implies an error or failure.

Reply:
I agree with @alejandro-colomar here. It's pretty much the same output you would get with the `perror` calls as they are in this file right now as well.